framework/mesh/io/vtk_io.cc

```diff
@@ -76,12 +76,11 @@ CreateCellFromVTKPolyhedron(vtkCell* vtk_cell)
 
   switch (sub_type)
   {
-    // The cell vertex ids in VTK is the same as in OpenSn, so we don't
-    // need to remap the vertices. We do however need to remap the faces.
+    // The cell vertex ids and faces can be preserved for hexahedrons
```

Comment:
I like the part of the old comments that stated "in VTK is the same as in OpenSn". I think we should keep this. It may help a subsequent reader.